modules/c++/nitf/include/nitf/IOHandle.hpp

```diff
@@ -46,14 +46,6 @@ class IOHandle : public IOInterface
 
 public:
 
-    /*!
```

Comment:
Oh I hadn't noticed that it was previously commented out - yeah, fine to remove this